src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -54,13 +54,19 @@ function ttn_svd(
   maxdim::Int=typemax(Int),
   cutoff=eps(real(coefficient_type)) * 10,
 )
-  sites = deepcopy(sites0)
+  # fermionic and non-fermionic QNIndices have opposite default index direction
+  # which causes problems when using the autofermion sign with non-fermionic QNIndex
+  # ToDo: remove conditional when this is addressed
+  linkdir_ref = ITensors.using_auto_fermion() ? ITensors.In : ITensors.In
+
+  sites = deepcopy(sites0)  # deepcopy because of inplace modification to handle internal indices 
```

Comment:
I think just `copy(sites0)` should be sufficient.

Reply:
i do a push! to edit the internal sites, and I verified that it breaks because of that. I can make it work with copy if you prefer.

Reply:
What breaks? We should only use `copy` instead of `deepcopy` if it works, and if it works then I would prefer `copy`.